frontend/src/components/Menu/DropdownMenu.stories.tsx

```diff
@@ -0,0 +1,204 @@
+        <>
+          <DropdownMenuButton isActive={isOpen} variant={variant} size={size}>
+            {children}
+          </DropdownMenuButton>
+          <DropdownMenuList>
+            <DropdownMenuItem>Last updated</DropdownMenuItem>
+            <DropdownMenuItem>Date created</DropdownMenuItem>
+            <DropdownMenuItem>Name</DropdownMenuItem>
+          </DropdownMenuList>
+        </>
+      )}
+    </DropdownMenu>
+  )
+}
+
+const DropdownMenuGroupTemplate: Story<DropdownMenuGroupTemplateProps> = ({
+  variant,
+}) => {
+  return (
+    <SimpleGrid
+      columns={2}
+      spacing={8}
+      templateColumns="max-content auto"
+      alignItems="center"
+    >
+      <Text>Closed Menu</Text>
+      <SimpleGrid
+        columns={2}
+        spacing={8}
+        templateColumns="min-content auto"
+        alignItems="center"
+      >
+        <DropdownMenuTemplate variant={variant} size={'normal'}>
+          Menu
+        </DropdownMenuTemplate>
+        <DropdownMenuTemplate variant={variant} size={'stretch'}>
+          Menu Stretch
+        </DropdownMenuTemplate>
+      </SimpleGrid>
+      <Text>Open Menu</Text>
+      <SimpleGrid
+        columns={2}
+        spacing={8}
+        templateColumns="min-content auto"
+        alignItems="center"
+      >
+        <DropdownMenuTemplate variant={variant} size={'normal'} isOpen>
+          Menu
+        </DropdownMenuTemplate>
+        <DropdownMenuTemplate variant={variant} size={'stretch'} isOpen>
+          Menu Stretch
+        </DropdownMenuTemplate>
+      </SimpleGrid>
+    </SimpleGrid>
+  )
+}
+export const Default = DropdownMenuTemplate.bind({})
+Default.args = {
+  variant: 'outline',
+  size: 'normal',
+  children: 'Menu Default',
+}
+
+export const Outline = DropdownMenuGroupTemplate.bind({})
+Outline.args = { variant: 'outline' }
+
+export const Clear = DropdownMenuGroupTemplate.bind({})
+Clear.args = { variant: 'clear' }
+export const Playground: Story = ({ variant, size }) => {
+  const toast = useToast()
+  return (
+    <Box>
+      <DropdownMenu>
+        {({ isOpen }) => (
+          <>
+            <DropdownMenuButton isActive={isOpen} variant={variant} size={size}>
+              EXPORT
+            </DropdownMenuButton>
+            <DropdownMenuList>
+              <DropdownMenuItem
+                onClick={() => {
+                  toast({
+                    status: 'success',
+                    description: 'Successfully downloaded',
+                    isClosable: true,
+                    duration: 2000,
+                  })
+                }}
+              >
+                CSV only
+              </DropdownMenuItem>
+              <DropdownMenuItem
+                onClick={() => {
+                  toast({
+                    status: 'success',
+                    description: 'Successfully downloaded',
+                    isClosable: true,
+                    duration: 2000,
+                  })
+                }}
```

Comment:
need to wrap in chakra theme provider

Reply:
thanks - to simplify, i've used alert instead